rtgui/cropwindow.cc

```diff
@@ -272,210 +269,209 @@ void CropWindow::buttonPress (int button, int type, int bstate, int x, int y)
 {
 
     bool needRedraw = true;  // most common case ; not redrawing are exceptions
+    EditSubscriber *editSubscriber = iarea->getCurrSubscriber();
```

Comment:
I see that the code is older, I just meant that if we touch stuff anyway, we can modify as if we would have written it today.